### PR TITLE
ONEUP-6027: fix service name

### PR DESCRIPTION
**Pull request by yolch-yolchyan**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -220,7 +220,7 @@ runs:
       if: ${{ steps.infrastructure-changes.outputs.terraform == 'true' && inputs.staging == 'true' }}
       working-directory: terraform
       run: |
-        terraform init -backend-config="key=staging/ecs_service_theme/terraform.tfstate"
+        terraform init -backend-config="key=staging/ecs_service_${{ env.SERVICE_NAME }}/terraform.tfstate"
```
Comment (catalin-cretu) [inline action.yml:223]: I think a search and replace is safer
```suggestion
        staging_env_key=$(sed -n 's/^.*production\/\(.*\)\"$/staging\/\1/p' backend.tf)
        terraform init -backend-config="key=${staging_env_key}"
```

**Review by catalin-cretu — APPROVED** [review]

**Review by querwurzel — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (yolch-yolchyan) [reply action.yml:223]: I think we will force consistency in this way to have `ecs_service_` as prefix

Comment (catalin-cretu) [reply action.yml:223]: Then it's awkward for theme-service because we have to use `ecs_service_theme_service`

Comment (yolch-yolchyan) [reply action.yml:223]: hmmm, I'll create a ticket to fix theme service. I'd say all our services should ideally have -service at the end

**Pull request merged**